src/Handler/PaymentNotificationHandler.php

```diff
@@ -72,9 +72,7 @@ public function treat(Generic $request, IVerifiableAPIResource $paymentResource,
 
         $this->entityManager->refresh($request->getFirstModel());
 
-        if (PayPlugApiClientInterface::STATUS_CREATED === $request->getFirstModel()->getDetails()) {
-            $lock->release();
-
+        if ($details['status'] === PayPlugApiClientInterface::STATUS_ABORTED) {
```

Comment:
Yon don't need to release the lock here ?

Reply:
added it back, but it should be released on class destruct anyway